### ISSUE TITLE
Species authorship missing but available with search API

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5533]
**Species authorship missing but available with search API**

This issue was originally reported on the GBIF helpdesk.

The authorship for this species is available in the search API response: https://api.gbif.org/v1/species/search?q=pieris%20napi%20napi&datasetKey=d7dddbf4-2cf0-4f39-9b2a-bb099caae36c (it shows as "(Linnaeus, 1758) ") but it isn't showing in this other API call https://api.gbif.org/v1/species/5714915/ nor in the species page.

-----

Github user: @ManonGros
User: [See in registry](https://www.gbif.org/api/feedback/user/c935ebbb6fe37641a61a7002de6cecc5:43debb50c75282526d2cb0645dd8bc42535532ef0ef5f08395ec3120a4abfcb223c8a6a22415a99e9d206be84da31cfa2c1499f2411632be33a5f98d5f2be160e4a4ad59accfb762fdff1b65556c347e3eeb56fb38a4355e5aa65f2797ee7af9702cf900df33db7c01294e183873bf5e06992e5a589dcd10e97a892508d9ebdb2249ca8a24f6b6edd617729f9bfa09f6) - [Send email](https://www.gbif.org/api/feedback/user/mailto/c935ebbb6fe37641a61a7002de6cecc5:43debb50c75282526d2cb0645dd8bc42535532ef0ef5f08395ec3120a4abfcb223c8a6a22415a99e9d206be84da31cfa2c1499f2411632be33a5f98d5f2be160e4a4ad59accfb762fdff1b65556c347e3eeb56fb38a4355e5aa65f2797ee7af9702cf900df33db7c01294e183873bf5e06992e5a589dcd10e97a892508d9ebdb2249ca8a24f6b6edd617729f9bfa09f6)
System: Safari 18.0.1 / Mac OS X 10.15.7
Referer: https://www.gbif.org/species/5714915
Window size: width 1728 - height 926
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-10-28T12:17:37.980Z',to:'2024-10-28T12:23:37.980Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-10-28T12:17:37.980Z',to:'2024-10-28T12:23:37.980Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by mdoering** [comment]
strange indeed. It is an autonym though (species epithet = subspecific epithet), in which case the backbone should not have any authorship.

**Comment by mdoering** [comment]
there is no authorship in the database records, just ES

**Comment by ManonGros** [comment]
@mdoering could this issue be related? https://github.com/gbif/backbone-feedback/issues/558

**Comment by mdoering** [comment]
yes indeed! I just have no clue why that is - I cannot see any wrong code. Puzzled
